core/storage/opportunity/gae_models.py

```diff
@@ -0,0 +1,131 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Models for Oppia users."""
+
+from core.platform import models
+
+from google.appengine.datastore import datastore_query
+from google.appengine.ext import ndb
+
+(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+
+
+class ExplorationOpportunitySummaryModel(base_models.BaseModel):
+    """Summary of translation and voiceover opportunities in an exploration.
+
+    The id of each instance is the id of the corresponding exploration.
```

Comment:
Where is this implemented? Shouldn't you write a create() function which does this?

Reply:
> which does this?

I'm not sure what we are expecting the `create` function to do? Can you please explain? Thanks! :)

Reply:
Where is the id of the model set is my question.... Shouldn't that be in the model class.

Reply:
Overe [here](https://github.com/oppia/oppia/pull/7109/files#diff-fce292386c105c085b26503b85de08f2R67), is that helpful?